runbotics/runbotics-orchestrator-ui/src/src-landing/components/BlogHeadImage/BlogHeadImage.module.scss

```diff
@@ -0,0 +1,63 @@
+@import '#src-landing/scss/main.scss';
+
+$BACKGROUND-HEIGHT: 400px;
+$BACKGROUND-HEIGHT-MD: 280px;
+$BACKGROUND-HEIGHT-S:225px;
+$BACKGROUND-HEIGHT-XS: 61px;
+
+
+.image {
+
+    position: relative;
+    margin-top: $HEADER_HEIGHT;
+    width: 100%;
+    height: $BACKGROUND-HEIGHT;
+    background: url('/images/banners/blog-header.png') no-repeat;
+    background-position: center;
+    background-size: cover;
+
+    &::before {
+        content: '';
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        z-index: $header-z-index;
+        background: transparent url('/images/blog/mask.svg');
+        mix-blend-mode: color;
+        opacity: 1;
+    }
+    
+    &::after{
+        @include font-face-base;
+        position: absolute;
+        z-index: 1;
+        content: "RunNews";
+        bottom: -40px;
+        right: 70px;
+        font-size: 17em;
+        color: white;
+        font-weight: $font-weight-semi-bold;
+        letter-spacing: -0.08em;
+        mix-blend-mode: soft-light;
+    }
+
+    @media (max-width: $breakpoint-md) {
+            height: $BACKGROUND-HEIGHT-MD;
+            font-size: 0.6em;

```

Comment:
I'm not sure about using the em when they base on the parent's font size. this may be the overreaction but personally i'd use rem or px

just fyi there is _spacing.scss file